actors/builtin/market/market_actor.go

```diff
@@ -301,7 +285,7 @@ func (a Actor) VerifyDealsOnSectorProveCommit(rt Runtime, params *VerifyDealsOnS
 		}
 
 		for _, dealID := range params.DealIDs {
-			deal, err := states.Get(dealID)
+			deal, _, err := states.Get(dealID)
```

Comment:
Not safe to ignore this `found` retval (it was already unsafe b/c deal could be nil).